primitives/src/lib.rs

```diff
@@ -76,6 +76,13 @@ pub type AuraPair = ed25519::Pair;
 /// The Ed25519 pub key of an session that belongs to an Aura authority of the chain.
 pub type AuraId = ed25519::Public;
 
+/// The Parachain crypto scheme defined via the keypair type.
+#[cfg(feature = "std")]
+pub type ParachainPair = ed25519::Pair;
```

Comment:
This will need changing almost instantly once the key type refactor is in to reflect.